ride_my_way/api/auth_views.py

```diff
@@ -0,0 +1,108 @@
+'''ímport dependancies'''
+import datetime
+from flask import jsonify, Blueprint, request, Flask
+from flask_jwt_extended import (
+    JWTManager, jwt_required, get_jwt_identity,
+    create_access_token, get_raw_jwt
+)
+from werkzeug.security import generate_password_hash, check_password_hash
+from ride_my_way import app, jwt
+from ride_my_way.api.models import RideMyWay
+
+blacklist = set()
+
+@jwt.token_in_blacklist_loader
+def check_if_token_in_blacklist(decrypted_token):
+    '''check if token is blacklisted'''
+    jti = decrypted_token['jti']
+    return jti in blacklist
+
+auth = Blueprint('auth', __name__)
+ride_my_way = RideMyWay()
+
+
+@app.route('/api/v1/auth/reset-password', methods=['POST'])
+def reset_password():
+    '''Function to reset user password'''
+    email = request.json.get('email')
+    for user in ride_my_way.users_list:
+        if email == user['email']:
+            user['password'] = generate_password_hash("Pass123")
+            return jsonify({
+                'message': "Password has been changed to Pass123. Please login and change it."
+            }), 201
+    return jsonify({'message': 'Email not found.'})
+
+@app.route('/api/v1/auth/register', methods=['POST'])
+def register():
+    '''Fuction to register a new user'''
+    try:
+        sent_data = request.get_json(force=True)
+        data = {
+            'user_id': len(ride_my_way.users_list) + 1,
+            'name': sent_data['name'],
+            'email': sent_data['email'],
+            'password': sent_data['password'],
+            'is_admin': False
+        }
+        if ride_my_way.check_email_for_login(data['email']):
+            return jsonify({'message': 'Email Exists'})
+        else:
+            if ride_my_way.user_data_validation(data):
+                return ride_my_way.user_registration(data), 201
+            else:
+                return jsonify(
+                    {'message': 'Please enter all the data in the correct format.'})
+    except BaseException:
+        return jsonify({'message': 'Please enter all the data required'})
+
+@app.route('/api/v1/auth/login', methods=['POST'])
```

Comment:
You could avoid repeating `/api/v1` in all your routes by registering your blueprint with `url_prefix='/api/v1'`